src/sentry/tasks/data_export.py

```diff
@@ -166,8 +137,10 @@ def create_writer(file, fields):
     return writer
 
 
-def get_file_name(export_type, custom_string, extension="csv"):
-    file_name = six.text_type("{}-{}.{}").format(export_type, custom_string, extension)
+def get_file_name(export_type, info_string, data_export_id, extension="csv"):
+    file_name = six.text_type(
+        "{}_{}_{}.{}".format(export_type, info_string, data_export_id, extension)
+    )
```

Comment:
This is to enforce the unique data_export_id suffix so that no files ever have the same name.